updated readme with the options for custom icons (#11)
sirensolutions/fontawesome-iconpicker: README.md

    searchInFooter: false, // If true, the search will be added to the footer instead of the title
    mustAccept: false, // only applicable when there's an iconpicker-btn-accept button in the popover footer
    selectedCustomClass: 'bg-primary', // Appends this class when to the selected item

>     replaceOriginalIcons: false //defaults to false, but can be updated as true when the icon picker is initialized. If true, replaces the original font awesome icons with the custom icons else appends to them

Reviewer: ```suggestion
    replaceOriginalIcons: false // If true, replaces the original font awesome icons with the custom icons, otherwise, appends the custom icons to the font awesome icons. Defaults to false.
```

What do you think, @akhileshbhatia?

Moved it around a bit to match https://github.com/sirensolutions/fontawesome-iconpicker/pull/11/files#diff-04c6e90faac2675aa89e2176d2eec7d8R102 but if you want to stick with yours, I'm happy with either
